src/main/java/edu/greenblitz/pegasus/commands/multiSystem/EjectFromShooter.java

```diff
@@ -0,0 +1,18 @@
+package edu.greenblitz.pegasus.commands.multiSystem;
+
+import edu.greenblitz.pegasus.RobotMap;
+import edu.greenblitz.pegasus.commands.funnel.ReverseRunFunnel;
+import edu.greenblitz.pegasus.commands.intake.roller.ReverseRunRoller;
+import edu.greenblitz.pegasus.utils.DigitalInputMap;
+import edu.wpi.first.wpilibj2.command.ParallelRaceGroup;
+import edu.wpi.first.wpilibj2.command.WaitUntilCommand;
+
+public class EjectFromShooter extends ParallelRaceGroup {//todo delete
```

Comment:
if the task is needed, finish it, and remove this comment, otherwise delete comment